src/app/settings/premium.component.html

```diff
@@ -1,15 +1,18 @@
-<div class="page-header">
+<div *ngIf="selfHosted" class="page-header">
+  <h1>{{ "subscription" | i18n }}</h1>
+</div>
+<div *ngIf="!selfHosted" class="tabbed-header">
```

Comment:
We are using a different pattern here than in `src/app/settings/user-subscription.component.html`. We should pick one.

Reply:
That's because of the pre-existing loading pattern that's in place for the `user-subscription` component. I didn't feel that it was within scope to try and restructure that for the sole purpose of supplying a header text.

Reply:
@vincentsalucci can you ensure we have a tech debt ticket for this in the SG pod?